chatkit-core/src/main/kotlin/com/pusher/chatkit/users/state/UserInternalType.kt

```diff
@@ -0,0 +1,13 @@
+package com.pusher.chatkit.users.state
+
+import com.pusher.chatkit.CustomData
+
+internal data class UserInternalType(
+    val id: String,
+    val createdAt: Long,
+    val updatedAt: Long,
+    val name: String?,
+    val avatarUrl: String?,
+    val customData: CustomData?,
+    var online: Boolean = false
```

Comment:
That definitely shouldn't be `var`. For now let's remove it as it won't be needed for current user and we'll need design on how to represent the presence in the state. We'll tackle it as part of MembersRepository.

Reply:
👍 good spot! I've removed the online var, you're right we can add it later!